commands/errors.go

```diff
@@ -9,43 +9,47 @@ import (
+	ErrIsRequired                                   = errors.New("is required")
+	ErrMustBePositive                               = errors.New("must be positive")
+	ErrMustBePositiveOrZero                         = errors.New("must be positive or zero")
+	ErrMustBeNegativeOrZero                         = errors.New("must be negative or zero")
+	ErrMustBeLessThan150                            = errors.New("must be less than 150")
+	ErrMustBeAtMost1M                               = errors.New("must be at most 1000000")
+	ErrMustBeAtMost100                              = errors.New("must be at most 100")
+	ErrMustBeWithinRange7                           = errors.New("must be between -7 and 7")
+	ErrIsNotValid                                   = errors.New("is not a valid value")
+	ErrIsNotValidNumber                             = errors.New("is not a valid number")
+	ErrIsNotSupported                               = errors.New("is not supported")
+	ErrIsUnauthorised                               = errors.New("is unauthorised")
+	ErrCannotAmendToGFA                             = errors.New("cannot amend to time in force GFA")
+	ErrCannotAmendToGFN                             = errors.New("cannot amend to time in force GFN")
+	ErrNonGTTOrderWithExpiry                        = errors.New("non GTT order with expiry")
+	ErrGTTOrderWithNoExpiry                         = errors.New("GTT order without expiry")
+	ErrIsMismatching                                = errors.New("is mismatching")
+	ErrReferenceTooLong                             = errors.New("reference is too long")
+	ErrNotAValidInteger                             = errors.New("not a valid integer")
+	ErrNotAValidFloat                               = errors.New("not a valid float")
+	ErrMustBeLessThan100Chars                       = errors.New("must be less than 100 characters")
+	ErrMustNotExceed20000Chars                      = errors.New("must not exceed 20000 characters")
+	ErrShouldBeHexEncoded                           = errors.New("should be hex encoded")
+	ErrSignatureNotVerifiable                       = errors.New("signature is not verifiable")
+	ErrInvalidSignature                             = errors.New("invalid signature")
+	ErrUnsupportedAlgorithm                         = errors.New("unsupported algorithm")
+	ErrEmptyBatchMarketInstructions                 = errors.New("empty batch market instructions")
+	ErrIsNotValidVegaPubkey                         = errors.New("is not a valid vega public key")
+	ErrIsNotValidEthereumAddress                    = errors.New("is not a valid ethereum address")
+	ErrIsNotValidEthereumAbi                        = errors.New("is not valid ethereum abi")
+	ErrIsNotValidEthereumMethodName                 = errors.New("is not valid ethereum method name")
+	ErrNoEthereumOracleNormalisers                  = errors.New("one or more normalisers are required")
+	ErrEthereumOracleRequiredConfirmationIsNegative = errors.New("required confirmations is negative")
```

Comment:
`ErrEthereumOracleRequiredConfirmationIsNegative` is unused (are we missing that validation) and anyway we can use `ErrMustBePositive` instead?

and I don't think we need `ErrNoEthereumOracleNormalisers` when we can use `ErrIsRequired`?

Reply:
sorted